react/BottomSheet/BottomSheet.jsx

```diff
@@ -70,6 +79,7 @@ const BottomSheet = ({ toolbarProps, settings, children }) => {
   const [peekHeights, setPeekHeights] = useState(null)
   const [bottomSpacerHeight, setBottomSpacerHeight] = useState(0)
   const [initPos, setInitPos] = useState(mediumHeight)
+  const isImmersive = Boolean(document.querySelector('.flagship-route-home'))
```

Comment:
nit: personnally I would have tested that feature

Reply:
est-ce qu'on peut avoir un terme plus explicite ? Quand est-ce qu'on a la classe `flagship-route-home` ? C'est quand on est sur la home de l'app amirale ? Il n'y a pas une meilleure façon de faire que de s'appuyer sur une classe css ? Un id peut-être plus ?

Reply:
Nouvelle API plus claire utilisée ✅

Reply:
`expect(container.querySelector('.flagship-route-home')).toBeInTheDocument()`